.gitignore

```diff
@@ -19,3 +19,7 @@ dist/
 
 # direnv
 .envrc
+
+# GoLand Project Files
```

Comment:
Must admit I'm not a fan of pushing these ignores to individual repos. That's probably something that should go in your user's (or system's) global gitignore config. If you don't mind I'll drop this commit from the PR.